### PR TITLE
[FEATURE] Add ./start.sh file

### PR DESCRIPTION
**Pull request by Kanti**

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  case ${key} in
+     start)
+        startFunction pull
+        startFunction build
+        startFunction up
+        startFunction login
+        return
+        ;;
+     up)
+        docker-compose -f docker-compose.dev.yml up -d
```
Comment (Kanti) [inline start.sh]:
```suggestion
        docker-compose -f docker-compose.yml up -d
```

File: start.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        return
+        ;;
+     down)
+        docker-compose -f docker-compose.dev.yml down --remove-orphans
```
Comment (Kanti) [inline start.sh]:
```suggestion
        docker-compose -f docker-compose.yml down --remove-orphans
```

File: start.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        return
+        ;;
+     login)
+        docker-compose -f docker-compose.dev.yml exec -u application web bash
```
Comment (Kanti) [inline start.sh]:
```suggestion
        docker-compose -f docker-compose.yml exec -u application web bash
```

File: start.sh [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        return
+        ;;
+     *)
+        docker-compose -f docker-compose.dev.yml "${@:1}"
```
Comment (Kanti) [inline start.sh]:
```suggestion
        docker-compose -f docker-compose.yml "${@:1}"
```

File: start.sh [hunk 5]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+function startFunction {
```
Comment (ChrisB9) [inline start.sh]: possibility to run composer install yarn and yarn build quickly would be amazing

Comment (Kanti) [reply start.sh]: this should be in another PR

**Pull request merged**